perception/autoware_probabilistic_occupancy_grid_map/src/pointcloud_based_occupancy_grid_map/pointcloud_based_occupancy_grid_map_node.cpp

```diff
@@ -270,6 +274,22 @@ void PointcloudBasedOccupancyGridMapNode::onPointcloudWithObstacleAndRaw()
       "debug/processing_time_ms", processing_time_ms);
     debug_publisher_ptr_->publish<autoware_internal_debug_msgs::msg::Float64Stamped>(
       "debug/pipeline_latency_ms", pipeline_latency_ms);
+
+    bool is_delay_within_range = (pipeline_latency_ms <= max_output_delay_ms_);
+    diagnostics_interface_ptr_->clear();
+    diagnostics_interface_ptr_->add_key_value(
+      "is_output_delay_within_range", is_delay_within_range);
+
+    std::stringstream message;
```

Comment:
Although it compiles, pre-commit should have warned about including the sstream header as well